### ISSUE TITLE
1.12.2 Bug Reports

### ISSUE THREAD

**Issue by PiggiesGoSqueal** [BenWoodworth/FastCraft#140]
## Description
1.) It doesn't support crafting by 64, 48, 32, and 24 for all items I've tried to craft so far. Currently that is paper. I have a full inventory of sugarcane and it doesn't work unless I do 16 or lower. Really slows down crafting.
2.) While crafting, it will sometimes randomly close out of the GUI and make a sound like an anvil I think. I wasn't done crafting but it kicked me out of the GUI for some reason. Really slows down crafting.

## Reproduction
- Take an inventory of sugarcane. Then set it to 64 crafting, it won't show. Go down to 16, craft really fast, it may close the GUI. It's inconsistent for the GUI closing.

## Output/Files
- Stack Trace: https://gist.github.com/
If this means errors in console, there are none.
- `/fcadmin debug`: https://gist.github.com/pinkpig3777/12c7e4760c78c80e4013dd083d2fa24d

- RELEVANT CONFIGS: https://gist.github.com/pinkpig3777/96bbd1cbaefc7523c12d6f7a2f3a0cd7

## Server Details
- FastCraft Version: v2.0.2
- Server Version: Spigot 1.12.2
- RELEVANT PLUGINS: https://gyazo.com/3481f83a52930888790ab60e06949b79

**Comment by SevastyanDark** [comment]
@pinkpig3777 Check first link. It is invalid

**Comment by pinkpig3777** [comment]
Sorry for the late reply. For some reason my email considers github messages as spam now. 

Unfortunately, this response is not bringing anything new to the table. I no longer have the stack trace and I tried to do it again but now I do not get an error in console for this. It did not close me out of the /craft GUI when I tried it but perhaps that's because I'm using an opped account? It still does not allow me to craft certain items (such as sugarcane) until I have it to 16 or less for the anvil icon.
